### PR TITLE
feat: Add save/restore state commands

### PR DESCRIPTION
**Pull request by artonge**

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -87,18 +87,35 @@ declare global {
...
+			 *
 			 * Query metadata for, and in behalf, of a given user
 			 *
 			 * @param user User whom metadata to query
 			 */
-			 getUserData(user: User): Cypress.Chainable<Cypress.Response<any>>
+			getUserData(user: User): Cypress.Chainable<Cypress.Response<any>>
+
+			/**
+			 *
```
Comment (skjnldsv) [inline lib/index.ts:105]:
```suggestion
```

**Review by skjnldsv — APPROVED** [review]

**Review by skjnldsv — CHANGES_REQUESTED** [review]
You are using the `nextcloud-cypress-tests_groupfolders` container name :see_no_evil:

**Comment by skjnldsv** [comment]
I remember I also did this already a month ago https://github.com/nextcloud-libraries/nextcloud-cypress/blob/main/lib/commands/snapshots.ts

Wanna add te data part to it instead?

**Comment by artonge** [comment]
Ah, I looked for that but couldn't find it. Let me upgrade it

**Comment by artonge** [comment]
Ah, I was looking in the master branch :see_no_evil:

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**